### PR TITLE
Adding automatic analyses from AugmentedNet v1.5.1

### PR DESCRIPTION
**Pull request by napulen**
This PR adds 1,296 automatic Roman numeral analyses for the `score.mxl` files located across the `Corpus` tree of the repository.

The analyses were generated by [AugmentedNet v1.5.1](https://github.com/napulen/AugmentedNet/), using the following script:

```python
import os
from pprint import pprint
import shutil

from AugmentedNet.inference import predict
from AugmentedNet.utils import tensorflowGPUHack
from tensorflow import keras


if __name__ == "__main__":
    wirpath = "When-in-Rome/Corpus"
    tensorflowGPUHack()
    model = keras.models.load_model("AugmentedNet.hdf5")
    success = []
    fail = []
    for root, _, files in os.walk(wirpath):
        for f in files:
            if f == "score.mxl":
                filepath = os.path.join(root, f)
                print(filepath)
                rn1 = f.replace(".mxl", "_annotated.xml")
                rn1path = os.path.join(root, rn1)
                rn2 = "analysis_automatic.txt"
                rn2path = os.path.join(root, rn2)
                if os.path.exists(rn2path):
                    success.append(rn2path)
                    continue
                try:
                    predict(model, filepath)
                except:
                    print("FAILED!")
                    fail.append(rn2path)
                    continue
                shutil.move(rn1path, rn2path)
                success.append(rn2path)
    pprint(success)
    pprint(fail)
``` 

Only a few files were unable to be processed by the model for various reasons (which I haven't explored). The unprocessed files are:

- When-in-Rome-master/Corpus/OpenScore-LiederCorpus/Warlock,_Peter/Saudades/3_Heracleitus/analysis_automatic.txt
- When-in-Rome-master/Corpus/OpenScore-LiederCorpus/Warlock,_Peter/Saudades/1_Along_the_Stream/analysis_automatic.txt
- When-in-Rome-master/Corpus/OpenScore-LiederCorpus/Lehmann,_Liza/Songs_of_Love_and_Spring/08_Love’s_Emblems/analysis_automatic.txt
- When-in-Rome-master/Corpus/OpenScore-LiederCorpus/Lehmann,_Liza/Songs_of_Love_and_Spring/10_The_Sapphire/analysis_automatic.txt
- When-in-Rome-master/Corpus/OpenScore-LiederCorpus/Parry,_Hubert/English_Lyrics_(1st_Set)/2_Good_night/analysis_automatic.txt

### CODE REVIEW

**Comment by MarkGotham** [comment]
HI @napulen, many thanks for this!

As you know, I think it would be great to have a set of augmented net analysis in this corpus so I definitely plan to accept and merge a version of this.

One edit we definitely need is for the file extension to match the type. As it stands, this PR actually provides musicXML (`.musicxml` ... 12 million lines of it) but files have the extension `.txt` (presumably for Romantext).

Were you planning to provide Romantext, or annotated musicXML, or both?

**Comment by napulen** [comment]
Hi @MarkGotham,

My bad, there is an error along the way. I intended RomanText.

Given that said, I released `v1.6.0` yesterday, with twice the accuracy on chord segmentation, so I'll re-compute these and share a new version of the **RomanText** files.

Thanks for noticing this!

**Comment by MarkGotham** [comment]
Sounds great.
And bravo on segmentation ... I hear it's quite important ([Gotham et al. 2021](https://doi.org/10.5281/zenodo.5676067)).

**Comment by napulen** [comment]
New batch coming up soon. Closing this PR.
